token/program-2022/src/extension/mod.rs

```diff
@@ -1101,6 +1105,10 @@ pub enum ExtensionType {
     GroupMemberPointer,
     /// Mint contains token group member configurations
     TokenGroupMember,
+
+    /// Mint allowing the minting and burn of confidential tokens
+    ConfidentialMintBurn = u16::MAX - 4269,
```

Comment:
Not missing anything 😄 I had used am purposefully high number initially to avoid possibly annoying rebase / merge conflicts with new extensions being merged into master and mapping to the same number